### ISSUE TITLE
Question about the OS-block

### ISSUE THREAD

**Issue by ShiinoMashiro** [Wensi-Tang/OS-CNN#17]
Hello,

I have read your paper and i have a few questions listed below about  the OS-block part:

Q1. "the first two layers of OS-block have prime-sized kernels from 1 to pk", why it is size of kernels, not size of receptive fileds?

Q2. "the two layers can cover all even number receptive field sizes", does this mean the RF is calculated just  by adding RF of layer1 and layer2 , will the batchnorm or relu part have influenced the size of RF?

I am looking forward to your reply.
Thanks!

**Comment by Wensi-Tang** [comment]
Hello, thanks for your attention!
Q1: For one layer, the receptive field sizes are the same as the kernel sizes. But for multiple inception layers, the receptive field sizes will be the combination of kernel sizes in each layer.
Q2: Based on the [definition (search "size of the region in the input that produces the feature") ](https://en.wikipedia.org/wiki/Receptive_field#In_the_context_of_neural_networks) of RF, the batch norm and ReLu will not influence the size.
